### ISSUE TITLE
onHighlightChange is not called in GWT

### ISSUE THREAD

**Issue by Spikatrix** [payne911/PieMenu#7]
PieMenu version 4.2.0 doesn't fire the `onHighlightChange` callback in GWT. I tested this on libGDX version `1.9.10`, `1.9.11` as well as `1.9.12-SNAPSHOT`. It works as expected in Desktop (LWJGL3)

I used the code in the [ClickDrag example](https://github.com/payne911/PieMenu/blob/master/src/test/java/com/payne/games/piemenu/codeExamples/ClickDrag.java) for testing it out.

I also tested it on PieMenu version 4.1.0 which worked as expected in both GWT as well as LWJGL3

**Comment by payne911** [comment]
Have you tested if the other Listeners work? (Is it *only* `onHighlightChange`?)

**Comment by Spikatrix** [comment]
I've experienced the issue on `onHighlightChange` (`onHoverChange` probably has the issue as well). `ChangeListener` works without issues.

Also I just noticed this error in the console:

```
Uncaught Error: java.lang.RuntimeException: Couldn't find Type for class 'com.payne.games.piemenu.PieMenu$PieMenuHighlightChangeEvent'
    at iZc_g$.DYc_g$ [as createError_0_g$] (Throwable.java:120)
    at iZc_g$.NYc_g$ [as initializeBackingError_0_g$] (Throwable.java:112)
    at iZc_g$.wYc_g$ (Throwable.java:66)
    at iZc_g$.aZc_g$ (Exception.java:29)
    at new iZc_g$ (RuntimeException.java:29)
    at Bsg_g$ (ReflectionCache.java:38)
    at rod_g$ (ClassReflection.java:135)
    at $gd_g$.ahd_g$ [as findConstructor_0_g$] (ReflectionPool.java:49)
    at new $gd_g$ (ReflectionPool.java:39)
    at sfd_g$ (Pools.java:29)
    at rfd_g$ (Pools.java:38)
    at tfd_g$ (Pools.java:48)
    at N5j_g$.b6j_g$ [as highlightIndex_0_g$] (PieMenu.java:644)
    at N5j_g$.c6j_g$ [as highlightSliceAtStage_0_g$] (PieMenu.java:662)
    at F6j_g$.K6j_g$ [as touchDragged_3_g$] (PieMenu.java:757)
    at F6j_g$._5b_g$ [as handle_3_g$] (InputListener.java:72)
    at m6b_g$.n7b_g$ [as touchDragged_1_g$] (Stage.java:319)
    at Wh_g$.Ji_g$ [as handleEvent_0_g$] (DefaultGwtInput.java:691)
    at HTMLDocument.<anonymous> (DefaultGwtInput.java:548)
```

A bit of debugging showed that it crashes [here](https://github.com/payne911/PieMenu/blob/master/src/main/java/com/payne/games/piemenu/PieMenu.java#L644)

**Comment by payne911** [comment]
Alright, I have the fix. Can you try the `gwt-pool-fix`? https://github.com/payne911/PieMenu/tree/gwt-pool-fix

More specifically, those were the commits that fixed the issue (I got mixed up a bit when referencing the issue in the commit message):

1) https://github.com/payne911/PieMenu/commit/707e41c0fe3e8a54c35d3cfd8e344bc95185a08c
2) https://github.com/payne911/PieMenu/commit/bdbd25d643611b40337378338bead71de71f94a0

Expect a **release by Monday**.

**Comment by Spikatrix** [comment]
Your fix works on GWT! :smiley:

**Comment by payne911** [comment]
Great!

Before merging the built GitHub Pages, I want to try to find a way to activate a similar configuration for my GWT Launcher:

https://github.com/payne911/PieMenu/blob/master/src/test/java/com/payne/games/piemenu/DesktopLauncher.java#L10

As it is, the PieMenus displayed on the online demo are far from being as sharp as on the desktop application, so I'd love that to be fixed. If you have any ideas, feel free to share!

**Comment by payne911** [comment]
`v4.3.0` is out! https://github.com/payne911/PieMenu/releases/tag/4.3.0

And the WebDemo has been updated to use it: https://payne911.github.io/PieMenu/

Let me know if anything is wrong. :)

It most probably should have been `v4.2.1` because there weren't any *major* additions, but I realized that a bit too late and didn't feel like going back onto all the modifications in the different branches to fix that.
